app/models/aleph_hint.rb

```diff
@@ -0,0 +1,62 @@
+# Parses Aleph marcxml and creates Hints
+#
+require 'open-uri'
+class AlephHint
+  def initialize
+    @marcxml = marcxml
+  end
+
+  def marcxml
+    Nokogiri::XML(open(Rails.application.secrets.aleph_hint_source))
+  end
+
+  def records
+    @marcxml.xpath('//xmlns:record')
```

Comment:
(and if it did we'd get an exception and alerted... I'm not sure what you are getting at)